ephios/plugins/federation/templates/federation/external_event_list.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}
+    {% translate "External events" %}
+{% endblock %}
+
+{% block content %}
+    <h1>{% translate "External events" %}</h1>
+    {% for event in events %}
+        <div class="list-group-item list-group-item-action p-0 d-flex flex-row">
+            <div class="m-0 py-2 d-flex flex-column flex-lg-row-reverse justify-content-around flex-grow-0">
+                <div class="ps-lg-2 d-flex flex-row flex-lg-column justify-content-center event-list-status-icon">
+                        <span class="text-muted far fa-user ps-2"></span>
+                </div>
+            </div>
+            <a class="list-group-item-action py-2 event-list-item-link"
+               href="{{ event.signup_url }}?referrer={{ SITE_URL }}">
```

Comment:
why are you passing the SITE_URL here?

Reply:
because the host needs to know where to send the user to complete the OAuth2 flow

Reply:
Ahh of course. I was wandering why it was popping up after having authorized the host already